naked-function-macro/src/naked.rs

```diff
@@ -191,7 +196,11 @@ fn emit_foreign_mod(func: &ItemFn, attrs: &ParsedAttrs) -> ItemForeignMod {
         ..func.sig.clone()
     };
     let foreign_fn = ForeignItem::Fn(ForeignItemFn {
-        attrs: attrs.foreign_attrs.clone(),
+        attrs: {
+            let mut attrs_ = attrs.foreign_attrs.clone();
+            attrs_.append(&mut attrs.cfg.clone());
```

Comment:
```suggestion
            attrs_.extend(&attrs.cfg);
```

Reply:
I tried that earlier but I got a type error due to it expecting `Attribute` but getting `&Attribute`. It seems that using `extend_from_slice` worked though.